form.html

```diff
@@ -0,0 +1,159 @@
+            <li class="main-nav__item main-nav__item--active"><a href="form.html">Feedback Form</a></li>
+            <li class="main-nav__item">HTML Academy</li>
+          </ul>
+          <button class="main-nav__toggle main-nav__toggle--close"></button>
+          <a class="main-nav__link" href="index.html"><img src="" class="main-nav__logo" alt="Sedona logo"></a>
+        </nav>
+
+        <div calss="intro">
+          <ul class="intro-list">
+            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
+            <li class="intro-list__item intro-list__item--middle">Sedona</li>
+            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
+          </ul>
+        </div>
+      </header>
+      <main class="main-content">
+        <section class="preamble">
+          <div class="preamble__inner">
+            <h2 class="preamble__header">Leave your feedback</h2>
+            <p class="preamble__paragraph">Help our hotels to become better! Leave a comment about it, also please leave a feedback about the sights you visited</p>
+          </div>
+        </section>
+
+        <form class="form" action="" method="get">
+
+          <section class="form__section">
+            <h2 class="form__section-header">Introduce yourself:</h2>
+            <div class="form__name">
+              <label class="form__label-text" for="name">Name:</label>
+              <input class="form__input-text" id="name" placeholder="Name" name="name" type='text'>
+            </div>
+            <div class="form__surname">
+              <label class="form__label-text" for="surname">Surname:</label>
+              <input class="form__input-text" id="surname" placeholder="Surname" name="surname" type='text'>
+            </div>
+            <div class="form__middle-name">
+              <label class="form__label-text" for="second-name">Middle name:</label>
+              <input class="form__input-text" id="second-name" placeholder="Middle name" name="second-name" type='text'>
+            </div>
+          </section>
+
+          <section class="form__section">
+            <h2 class="form__section-header">Contact Information:</h2>
+            <div class="form__phone">
+              <label class="form__label-tel" for="phone">Contact number:</label>
+              <input class="form__input-tel" placeholder="Phone number" name="phone" type="tel">
+              <p class="form__paragraph--error">Error, double-check the number</p>
+            </div>
+            <div class="form__email">
+              <label class="form__label-email" for="email">Email:</label>
+              <input class="form__input-email" id="email" placeholder="user@domain.co.uk" name="email" type="email">
+            </div>
+          </section>
+
+          <section class="form__section">
+            <fieldset class="form__radio-list">
```

Comment:
question: Why do we need fieldset here, other form sections doesn't have it. What's going to be the purpose of it?

Reply:
It comes  for the MDN website: "The HTML <fieldset> element is used to group several controls as well as labels (<label>) within a web form." 
Also the tutorial "Interneting is hard" is using it.
But  the thing about it, is that is hard to style. So if you know how to not use it I will be happy to fallow the idea.

Reply:
You just don't have to use it. But I agree you need some sort of a wrapper around your checkboxes, it could be just div. Or you can also restyle fieldset if you want.